pgmpy/estimators/MirrorDescentEstimator.py

```diff
@@ -45,7 +46,7 @@ def _calibrate(self, theta, n):
         log_z = logsumexp(mu[clique].values)
```

Comment:
This would need a compat_fn:
```python
def logsumexp(arr):                                                                                                                                                                                                                                                                                                                                     
     if isinstance(arr, np.ndarray):                                                                                                                                                                                                                                                                                                                     
         return special.logsumexp(arr)                                                                                                                                                                                                                                                                                                                   
     else:                                                                                                                                                                                                                                                                                                                                               
         return torch.logsumexp(arr, 0)   
```

Reply:
Similar story here, `logsumexp(mu[clique].values)` reduces to a float which is compatible with `mu[clique] += np.log(n) - log_z`. `scipy.special.logsumexp` is also compatible with `torch.Tensor`:
```
import torch
from scipy.special import logsumexp

logsumexp(torch.Tensor([10, 10, 10]))
```
When axis=0, this reduces to a float.